Dockerfile

```diff
@@ -15,4 +15,6 @@ RUN pipenv install --system --deploy
 # the sources are copied as late as possible since they are likely to change often
 ADD geocoder_tester/ /geocoder_tester/
 
-CMD ["/run_qwant.sh", "/results", "http://bragi:4000/autocomplete"]
+ENV BRAGI_URL "http://bragi:4000/autocomplete"
+
+CMD ["/run_qwant.sh", "/results", ${BRAGI_URL}]
```

Comment:
Hum, does that really help to override the URL programmatically ?

Reply:
yes this way you can do: docker run -it -e BRAGI_URL=bob geocoder_tester

(and there is a default value to "http://bragi:4000/autocomplete"